internal/slicer/report.go

```diff
@@ -70,3 +69,33 @@ func (r *Report) Add(slice *setup.Slice, fsEntry *fsutil.Entry) error {
 	}
 	return nil
 }
+
+// Mutate updates the FinalHash and Size of an existing path entry.
+func (r *Report) Mutate(fsEntry *fsutil.Entry) error {
+	relPath, err := r.sanitizePath(fsEntry.Path, fsEntry.Mode.IsDir())
+	if err != nil {
+		return fmt.Errorf("cannot add path: %w", err)
+	}
+
+	entry, ok := r.Entries[relPath]
+	if !ok {
+		return fmt.Errorf("path %q has not been added before", relPath)
```

Comment:
I think we should change it to something along the lines of:
```
cannot mutate path %q: not added previously in report
```

Reply:
```
cannot mutate path %q: no entries in report
```

What do you think about this?

Reply:
I like your version better